src/configContext/index.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import ConfigProvider from './configProvider'
+import ConfigConsumer from './configConsumer'
+
+export default {
+  Provider: ConfigProvider,
+  Consumer: ConfigConsumer,
+}
```

Comment:
why export both? We only need `ConfigConsumer` externally (the provider is only used internally, I would not export it)

Reply:
Make sense. I'll make Provider only exportable inside it's own file but not in `gymnast.js`

Reply:
As I was working on removing this, I noticed (via Storybook) that when using `<ConfigProvider>` imported from a relative path alongside a component (i.e. `<Grid>`) imported from a `gymnast` import, that `<Grid>` will not use the same `React.createContext()` as provided from `<ConfigProvider>` when imported from a relative path.

This is only an issue in Storybook in the `Config Provider` section. However, this shouldn't be a problem in production, since we're only exporting `Consumer`, which should be using the same `React.createcontext()` as the `Provider` imported from `gymnast`.

Example:
```
// @flow
// storybook/stories/configProvider/columns.js

import * as React from 'react'
import { times } from 'lodash'
import { number } from '@storybook/addon-knobs'
import { Grid } from 'gymnast'
import { colors } from '../../shared'

// This is using a different `React.createContext()` than the imported
// Grid above. Therefore, Grid isn't going to consume the 'columns' that
// was passed through from this ConfigProvider, and instead, will used
// the default columns value.
import ConfigProvider from '../../../src/configContext/configProvider'

export default () => {
  const columns = number('Columns', 10, { range: true, min: 1, max: 24 })
  const items = number('Items', 20, { range: true, min: 1, max: 48 })

  return (
    <ConfigProvider columns={columns}>
      <Grid padding="0 L/2">
        {times(items, key => (
          <Grid key={key} padding="0 L/2 L" size={1}>
            <Grid
              style={colors.colors1}
              padding="L/2"
              align="center"
              justify="center"
            >
              {key + 1}
            </Grid>
          </Grid>
        ))}
      </Grid>
    </ConfigProvider>
  )
}
```

This appears to be a known issue via this [issue](https://github.com/facebook/react/issues/13346)

To alleviate the Storybook problem, I can remove all imports from `gymnast` and just imported via relative path, like so:

```
// @flow
// storybook/stories/configProvider/columns.js

import * as React from 'react'
import { times } from 'lodash'
import { number } from '@storybook/addon-knobs'
import { colors } from '../../shared'
import ConsumerProvider from '../../../src/configContext/configProvider'
import Grid from '../../../src/grid'

export default () => {
  const columns = number('Columns', 10, { range: true, min: 1, max: 24 })
  const items = number('Items', 20, { range: true, min: 1, max: 48 })

  return (
    <ConsumerProvider columns={columns}>
      <Grid padding="0 L/2">
        {times(items, key => (
          <Grid key={key} padding="0 L/2 L" size={1}>
            <Grid
              style={colors.colors1}
              padding="L/2"
              align="center"
              justify="center"
            >
              {key + 1}
            </Grid>
          </Grid>
        ))}
      </Grid>
    </ConsumerProvider>
  )
}
```

Reply:
We could add a babel alias to it to make it a bit more convenient to use. If we define one (for stories only) that maps `gymnastSrc` to the `src` path we could then do something like `gymnastSrc/configContext/configProvider`

Reply:
to ensure the project is still tree-shakeable with [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) this `index.js` file needs to contain as default export the same value we export through `src/index.js`